backend/meta/collections/10017.chaos-engineering.yml

```diff
@@ -0,0 +1,13 @@
+id: 10017
+name: Chaos Engineering
+items:
+  - dastergon/awesome-chaos-engineering
```

Comment:
https://github.com/dastergon/awesome-chaos-engineering this should be removed?

Reply:
Yeah, I think I would remove it.

I found there were no other "awesome-*" projects collected.

Reply:
Updated! PTAL :)

Reply:
LGTM